packages/styles/src/components/button.scss

```
@@ -134,22 +134,48 @@
 }
 
 // Animated
+.btn-animated,
+.btn-animated-left {
```

With Bootstrap 5 we updated all classes that used "left" and "right" to "start" and "end", let's use the same wording here too: 
```suggestion
.btn-animated-start {
```